configure

```diff
@@ -4098,7 +4101,6 @@ case $_backend in
 	3ds)
 		;;
 	android)
-		append_var DEFINES "-DREDUCE_MEMORY_USAGE"
```

Comment:
This Android change is not related to the rest of the NDS changes

Reply:
Indeed, but as I am the de facto maintainer of the Android port and as this change is related to the commit before (the COMMON one), I took the liberty to include it.